emsdk

```diff
@@ -492,7 +501,10 @@ def unzip(source_filename, dest_dir, unpack_even_if_exists=False):
             parent_dir = os.path.dirname(fix_potentially_long_windows_pathname(final_dst_filename))
             if parent_dir and not os.path.exists(parent_dir):
               os.makedirs(parent_dir)
-            os.rename(fix_potentially_long_windows_pathname(tmp_dst_filename), fix_potentially_long_windows_pathname(final_dst_filename))
+            target = fix_potentially_long_windows_pathname(final_dst_filename)
+            if os.path.exists(target):
+              os.remove(target)
+            move_with_overwrite(fix_potentially_long_windows_pathname(tmp_dst_filename), fix_potentially_long_windows_pathname(final_dst_filename))
```

Comment:
can't the second arg to move_with_overwrite just be `target`? An that makes the check/remove on L505 redundant.

Reply:
Oh, oops, yes, those earlier lines are redundant now that I put the logic in a function. Somehow I forgot those. Fixed now.